surveys/2024-annual-survey/translations/de.md

```diff
@@ -546,7 +546,7 @@ Type: select all that apply
 - Programmiersprachen und zugehörige Programme (einschließlich Compiler, IDEs, Standardbibliotheken usw.)
 - Robotik
 - Wissenschaftliches und/oder numerisches Rechnen
-- Serverseitige oder &quot;Backend&quot;-Anwendung
+- Serverseitige oder "Backend"-Anwendung
```

Comment:
Note: This was not strictly needed, it looks like in the survey they were properly displayed as `"`